### DISCUSSION TITLE
Filtering `created` field date only

### DISCUSSION THREAD

**Discussion by RA9** [pocketbase/pocketbase#1588] [Q&A] [answered]
Hi Boss,
I have been using pocketbase and it has been great. Thanks for the work, so far.

I want to know the best way to filter the `created` field using the current date or `new Date()`.
```js
const currentDate = new Date();
const isCheckedIn = (await locals.pb.collection('attendance').getList(1, 50, {
            filter: `created = "${currentDate}"`,
        }));
```

The above code doesn't seem to work because of the time attached to the created. How can I compare the current date with the created date without adding the time?

**Comment by ganigeorgiev** [comment, marked as answer]
The date string need to be in the format `Y-m-d H:i:s.uZ`, aka. "2023-01-13 09:10:00.123Z" (_or part of it_).

If you don't want to specify a time and to target an entire day/period, you can use a "range" style filter, eg.:

```js
const result = await locals.pb.collection('attendance').getList(1, 50, {
    // or a more simpler generic form like `created > "2023-01-12" && created < "2023-01-14"`
    filter: `created >= "2023-01-13 00:00:00.000" && created <= "2023-01-13 23:59:59.999"`,
});
```

You can't pass directly a `Date` object to the filter. It needs to be converted to a properly formatted string first. Also please note that dates are stored in UTC.

**Reply by RA9** [reply]
Thanks, it worked.
